docs/csharp/tour-of-csharp/statements.md

```diff
@@ -38,79 +39,79 @@ The `using` statement is used to obtain a resource, execute a statement, and the
 
 The following lists the kinds of statements that can be used, and provides an example for each.
 
-***Local variable declaration***
+* Local variable declaration:
 
 [!code-csharp[Declarations](../../../samples/snippets/csharp/tour/statements/Program.cs#L9-L15)]
```

Comment:
Written like this, each bullet point is its own list (`<ul>`), which is not semantically correct. To make it all one list, as it should be, each snippet should be indented by one space.